internal/cmd/config/config.go

```diff
@@ -379,6 +383,36 @@ func Parse(d string) (*Config, error) {
 						reflect.TypeOf(t).String())
 				}
 			}
+			if result.Controller.Database.MaxIdleConnectionsRaw != nil {
+				switch t := result.Controller.Database.MaxIdleConnectionsRaw.(type) {
+				case string:
+					maxIdleConnectionsString, err := parseutil.ParsePath(t)
+					if err != nil {
+						return nil, fmt.Errorf("Error parsing database max idle connections: %w", err)
+					}
+					result.Controller.Database.MaxIdleConnections, err = strconv.Atoi(maxIdleConnectionsString)
+					if err != nil {
+						return nil, fmt.Errorf("Database max idle connections value is not an int: %w", err)
+					}
+				case int:
+					result.Controller.Database.MaxIdleConnections = t
+				default:
+					return nil, fmt.Errorf("Database max idle connections: unsupported type %q",
+						reflect.TypeOf(t).String())
+				}
+			} else {
+				result.Controller.Database.MaxIdleConnections = -1
```

Comment:
I don't know if this is a well established pattern (using -1 to indicate "unset"), but if it's not, what do you think about using a pointer instead? Then we could make the type unsigned too.

Reply:
Yea I am not sure. I don't think we have done `-1` in the past. I thought about using a pointer but it felt cumbersome, but maybe it is cleaner. The main thing I want to avoid is accidentally setting this to `0` since that is different from the default value in `sql.DB`.

Reply:
I think if it's all the same to you I'd prefer a pointer, but it seems something that maybe @jefferai or @jimlambrt might have opinions about too 😁.